src/pages/Write/Write.js

```diff
@@ -0,0 +1,258 @@
+import React, { useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { flexSet } from '../../styles/Variable';
+
+function Write({ theme, props }) {
+  const [file, setFile] = useState('');
+  const [previewURL, setPreviewURL] = useState('');
+
+  const handleFileInput = e => {
+    e.preventDefault();
+    let reader = new FileReader();
+    let file = e.target.files[0];
+    reader.onloadend = () => {
+      setFile(file);
+      setPreviewURL(reader.result);
+    };
+    reader.readAsDataURL(file);
+  };
+  const handlePost = () => {};
+
+  return (
+    <Wrapper>
+      <Filter>
+        <SelectContainer top>
+          <Select top name="category">
+            <Option grey value="" selected disabled>
+              주거형태
+            </Option>
+            <Option value="원룸&오피스텔">원룸&오피스텔</Option>
+            <Option value="아파트">아파트</Option>
+            <Option value="단독주택">단독주택</Option>
+          </Select>
+        </SelectContainer>
+        <SelectContainer top>
+          <Select top name="space">
+            <Option grey value="" selected disabled>
+              평수
+            </Option>
+            <Option value="10평미만">10평 미만</Option>
+            <Option value="10평대">10평대</Option>
+            <Option value="20평대">20평대</Option>
+            <Option value="30평대">30평대</Option>
+            <Option value="40평대">40평대</Option>
+            <Option value="50평이상">50평 이상</Option>
+          </Select>
+        </SelectContainer>
+        <SelectContainer top>
+          <Select top name="style">
+            <Option grey value="" selected disabled>
+              스타일
+            </Option>
+            <Option value="모던">모던</Option>
+            <Option value="북유럽">북유럽</Option>
+            <Option value="내추럴">내추럴</Option>
+          </Select>
+        </SelectContainer>
+      </Filter>
+      <WriteContainer>
+        {file ? (
+          <Preview>
+            <ProfilePreview src={previewURL} alt="uploaded"></ProfilePreview>
+          </Preview>
+        ) : (
+          <PhotoContainer>
+            <UploadInput
+              type="file"
+              name="photo"
+              id="photo"
+              accept="image/*"
+              onChange={e => handleFileInput(e)}
```

Comment:
사소하지만 event 객체는 자동으로 인자로 넣어주면서 호출하기때문에 
```suggestion
              onChange={handleFileInput}
```
이 방식으로만해도 충분해보입니다!
인라인으로 함수를 작성하면 새로운 함수를 다시 한번 생성하는 연산을 해야되기때문에 비효율적입니다!